app/experimenter/experiments/models/nimbus.py

```diff
@@ -19,7 +19,14 @@
 
 class NimbusExperimentManager(models.Manager):
     def launch_queue(self, application):
-        return self.filter(status=NimbusExperiment.Status.REVIEW, application=application)
+        return self.filter(
+            Q(status=NimbusExperiment.Status.DRAFT)
+            | Q(status=NimbusExperiment.Status.PREVIEW),
```

Comment:
Wait no what?  We don't launch preview experiments?

Reply:
Shouldn't this whole thing just be `.filter(status=DRAFT, publish_status=APPROVED, application=application)`?

Reply:
@jaredlockhart How does an experiment in Preview lifecycle status get pushed to Live?

Reply:
> or forward to launch (DRAFT, APPROVED)

So it actually goes out of Preview and back to Draft before going to Live?

Reply:
Huh yeah maybe we've misunderstood something then, can't a launch be requested while the experiment is in `PREVIEW`?

Reply:
Right, and a launch is setting it to `(DRAFT, APPROVED)`, regardless of whether it's in `(DRAFT, IDLE)` or `(PREVIEW, IDLE)`

Reply:
When it goes from `(PREVIEW, IDLE)` to `(DRAFT, APPROVED)`, 

1) it's no longer in preview so it'll get pulled out of the preview collection
2) the launch task will pick it up from `(DRAFT, APPROVED)` launch list

Reply:
Maybe I should've made the preview diagrams a little more explicit....

Reply:
Okay. Well, changing this back does break the Request Launch flow (just tested), so I'm going to go look at what else needs updating to get a Preview experiment's launch request change to show up in RS

Reply:
Yeah make sure the frontend mutation is setting `(DRAFT, APPROVED)` when you're in preview and click the launch, and that the launch_queue is just looking for `(DRAFT, APPROVED)`

Reply:
Think I found it: https://github.com/mozilla/experimenter/blob/issues/4850/app/experimenter/nimbus-ui/src/components/PageRequestReview/index.tsx#L103:L103

Needs to be 

```js
{
  status: NimbusExperimentStatus.DRAFT,
  publishStatus: NimbusExperimentPublishStatus.APPROVED,
},
```

Reply:
Oh you commented that before me. 🙃

Reply:
Right exactly it should be that for the launch button regardless of whether you're currently in preview or not

Reply:
More out-loud thinking through this...

In the next stage of review flow, launching from preview will start with a transition to DRAFT / REVIEW.

That will un-publish the experiment from the preview collection while the review is pending. (That seems a little confusing to me but it does simplify some things)

If review is rejected, the experiment lands back in DRAFT / IDLE status.

If timed out, the experiment lands back in DRAFT / REVIEW status.

Reply:
Exactly yep 👍

Reply:
The experiment is only in the preview collection when it's in the preview state.  When it's being reviewed, it's not in preview.